packages/create-vite/template-react/package.json

```
@@ -6,21 +6,23 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint .",
```

This is important IMO, the default of having warnings be ok is bad
```suggestion
    "lint": "eslint . --max-warnings 0",
```

---

This feels more like a personal preference I think so I removed it. If we disallow any warnings, doesn't it make them as important as errors?

---

Is there really people that like to see committed code to the main branch with warning in their IDE?

CRA had a great philosophy about that: eveything should be treated, but warnings are for things that would not break the app (like bad pattern) and errors are for things that will break the app (bad import, uninitialized variable, ...)

---

Some people are fine with that, that's the point of warnings. I think it's a preference thing if they mind that. Some eslint errors are also used for _bad patterns_.